python/lsst/obs/sdss/convertOpECalib.py

```diff
@@ -50,5 +50,6 @@ def getEParams(self, ccdName):
 #-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-
 
 if __name__ == "__main__":
-    sc = SdssCameraState("/lsst7/stripe82/dr7/opfiles", "opConfig-50000.par", "opECalib-50000.par")
-    print [(i, ep.getGain(), ep.getReadNoise(), ep.getSaturationLevel()) for i, ep in sc.getEParams("g2")]
+    sc = SdssCameraState(os.path.expandvars("$OBS_SDSS_DIR/etc"), "opConfig-50000.par", "opECalib-50000.par")
```

Comment:
This should use lsst.utils.getPackageDir("obs_sdss") instead of depending on an environment variable that may go away.